### ISSUE TITLE
Can you share pretrained weights file

### ISSUE THREAD

**Issue by k5iogura** [ArtyZe/yolo_segmentation#20]
Hi, 
ThanX great work as such this project.
I have started reproducing your great result, but a no pretrained weight to check our result.

Can you send me pretrained weights file to below,

k5i.ogura.v40@gmail.com

As of now, we can not get good result of our project, but we try hardly to reproduct and will send you our result sometimes.

Best regards.

**Comment by ArtyZe** [comment]
@k5iogura 
I have updated readme and you can get then in dropbox

**Comment by k5iogura** [comment]
Thanx for your replay.

As of now, I have get 68.92AP via segmentationClass of VOC dataset.
Iteration 220610 lr=1e-7 loss=2868.56 5days with two GTX1080Ti, whooo,,,

By the way,
I set ground truth to 0 or 1 a pixel, here 1 if object exists else 0, so 2classes training.

If I got the other result, then report for you again.

Best Regards,

k5iogura from Japan.

2020/04/16 18:58、ArtyZe <notifications@github.com>

**Comment by ArtyZe** [comment]
@k5iogura 
Nice. Just like what Kaiming He said in paper "Rethinking ImageNet Pre-training", pretrain model is not important for model training. 
And what important for next step is instance segmentation, I have tried to optimize darknet's instance segementation code, but train failed :(

**Comment by ZafarFaraz2020** [comment]
Hey,
Can you share the pre-trained weights file? 
My email ID is zafaraz26@gmail.com

Thank you

**Comment by ArtyZe** [comment]
Hi,
    you can get all cfg and weights file from link without password:
    https://www.dropbox.com/sh/9wrevnyzwfv8hg7/AAA1MJElri9aROsjaPTxO5KCa?dl=0
    
    Best regards :)
| |
高扬
|
|
gaoyang917528@163.com
|
签名由网易邮箱大师定制
On 8/19/2020 10:08，ZafarFaraz2020<notifications@github.com> wrote：

Hey,
Can you share the pre-trained weights file?
My email ID is zafaraz26@gmail.com

Thank you
